internal/controllers/objectdeployments/objectdeployment_controller.go

```diff
@@ -106,25 +106,25 @@ func NewClusterObjectDeploymentController(
 		adapters.NewClusterObjectDeployment,
 		newGenericClusterObjectSet,
 		newGenericClusterObjectSetList,
+		isBootstrap,
 	)
 }
 
-func (od *GenericObjectDeploymentController) Reconcile(
-	ctx context.Context, req ctrl.Request,
-) (ctrl.Result, error) {
+func (od *GenericObjectDeploymentController) Reconcile(ctx context.Context, req ctrl.Request) (res ctrl.Result, err error) {
 	log := od.log.WithValues("ObjectDeployment", req.String())
 	defer log.Info("reconciled")
+
+	/*if od.isBootstrap && req.Name != "package-operator" {
+		return
+	}*/
```

Comment:
Since we have multi-component packages coming, we'll probably split the package-operator package into separate packages (for example putting the webhooks into a sub-component).

This will obviously not work with a single-hardcoded name, so I think we should use a labelSelector to select PKO components that the bootstrapper will reconcile.